README.md

```diff
@@ -34,8 +34,9 @@ If you are developing a project that uses new ECMAScript language features and m
 - Enables [class-properties](https://www.npmjs.com/package/babel-plugin-transform-class-properties) (stage 3)
 - Enables [object-rest-spread](https://www.npmjs.com/package/babel-plugin-transform-object-rest-spread) (stage 3)
 - Optionally enables React
+- Uses [add-exports](https://github.com/59naga/babel-plugin-add-module-exports) to get around [babel#2212](https://github.com/babel/babel/issues/2212)
```

Comment:
Can we have the link say `add-module-exports` instead? Just to match the plugin name.

Reply:
Fixed, my bad.